content/docs/learning-guide/tutorials/assets/physx-colliders.md

```diff
@@ -0,0 +1,162 @@
+
+8. Add a **PhysX Collider** component to the entity. With the entity selected, in **Entity Inspector**, choose **Add Component** and select **PhysX Collider** from the component list. The `.pxmesh` asset is automatically detected by the component. The PhysX Collider component's **Shape** property is set to `PhysicsAsset` and the **PhysX Mesh** property automatically references the `.pxmesh` product asset.
+
+9. With just a PhysX Collider component, this is a static entity. PhysX objects can collide with this entity, but the entity will not move or react to PhysX collisions or forces.
+
+{{< note >}}
+If you intend for the entity to be static, you should also activate the **Static** property in the **Transform** component of the entity. The **Static** property optimizes the entity, ensuring the best runtime performance for static entities.
+{{< /note >}}
+
+10. If you intend to create a kinematic or dynamic entity, you must add a **PhysX Rigid Body** component as well. With the entity selected, in Entity Inspector, choose **Add Component** and select PhysX Rigid Body from the component list. Ensure the **Static** property in the entity's Transform component is not active.
+
+{{< note >}}
+Adding the PhysX Rigid Body component makes the entity dynamic. If you choose the {{< icon simulate-physics.svg >}} simulation button, the entity will drop with gravity.
+
+If you want to create a kinematic entity that is animated through script, activate the **Kinematic** property in the PhysX Rigid Body component.
+{{< /note >}}
+
+## Decompose input meshes
+
+Assets that are composed of multiple meshes, or assets that have complex meshes, might require similarly complex PhysX collider assets. This is particularly true for kinematic and dynamic entities where triangle mesh colliders can't be used. In these scenarios, the input mesh can be decomposed into convex parts. Primitive colliders or convex colliders can be automatically generated and fitted to each part and processed as collider `.pxmesh` product assets.
+
+{{< note >}}
+Mesh decomposition doesn't alter the input mesh. Mesh decomposition is part of the process of generating and fitting collider assets. Mesh decomposition can only be used with primitive or convex colliders.
+
+Mesh decomposition in O3DE uses the [V-HACD Library](https://github.com/kmammou/v-hacd). Refer to the preceding link for more information and example images demonstrating decomposition of complex meshes.
+{{< /note >}}
+
+To use mesh decomposition, do the following:
+
+1. In Scene Settings, in the PhysX mesh group, choose either `Primitive` or `Convex` as the **Export As** property. 
+1. Activate the **Decompose Meshes** option.

```

Comment:
```suggestion
1. Enable **Decompose Meshes**.
```
* For consistency with my earlier edits, changed "activate" to "enable".
* To avoid confusion over whether to call this an "option", "property", or "parameter", I suggest dropping the noun at the end.